### DISCUSSION TITLE
docs/ninjutsu/up-ranking-ninja/leetcode-editor

### DISCUSSION THREAD

**Discussion by giscus** [MingGH/996-ninja-giscus#25] [General]
# docs/ninjutsu/up-ranking-ninja/leetcode-editor

学习，任何时候都要学！不学不行，你们想想，你毕了业带着女朋友北了漂，敲着代码还唱着歌，突然就被资本家压榨了……所以，没有资本家的日子才是好日子！

https://996.ninja/docs/ninjutsu/up-ranking-ninja/leetcode-editor

**Comment by wududu0920** [comment]
问题来了，让子弹飞什么时候申遗awa

**Comment by Clayblockunova** [comment]
Eclipse有无类似插件？
